src/Bicep.Core/Syntax/WildcardImportSyntax.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Bicep.Core.Navigation;
+using Bicep.Core.Parsing;
+
+namespace Bicep.Core.Syntax;
+
+public class WildcardImportSyntax : SyntaxBase, INamedDeclarationSyntax
```

Comment:
With `import * as foo from 'foo.bicep`, will all the members in `foo.bicep` be put under a namespace `foo` and can only be accessed via the namespace? If so, do we have plans to add support for wildcard imports without alias (`import * from 'foo.bicep'`) and default imports (`import 'foo.bicep'`)?

Reply:
Default imports is currently being blocked by provider imports, though.

Reply:
I'd prefer to hold off on both default imports and unaliased wildcards. Unaliased wildcards create a situation where your template can start failing to compile because something you're importing added a new symbol whose name clashes with something else defined in your template. Any case where a template starts breaking without changes to the source should be viewed with suspicion IMHO.

For default imports, I'm just not sure if there would be demand. It seems like something we can add later if users ask for it.